sdk/tables/azure-data-tables/src/serializers.cpp

```diff
@@ -0,0 +1,237 @@
+
+#include <azure/data/tables/serializers.hpp>
+
+using namespace Azure::Storage;
+using namespace Azure::Data::Tables;
+
+std::string const Serializers::CreateEntity(Models::TableEntity const& tableEntity)

```

Comment:
You didn't have to move them to .cpp, if you've only done that for not doing the "using namespace", but this will also work.

Alternatively, you could've put "using namespace Azure::Storage" inside of each of these inline functions.

Reply:
yeah but that looks ugly , using in a method grrrr ,